modules/300-prometheus/images/grafana-dashboard-provisioner/hooks/dashboard_provisioner.sh

```diff
@@ -47,6 +48,25 @@ function __main__() {
 
     title=$(slugify <<<${title})
 
+    dashboardUid=$(jq -rc '.definition | try(fromjson | .uid)')
+    check_dashboard=0
+
+    if [[ ! -s "${tmpFile}" ]]; then
+      echo "${dashboardUid}" >> "${tmpFile}"
+    else
+      for uid in $(cat "${tmpFile}"); do
+        if [[ "${uid}" == "${dashboardUid}" ]]; then
+          check_dashboard=1
```

Comment:
```suggestion
          check_dashboard=1
          break
```

You don't need to iterate further if you have found at least 1 duplicate